class CommandError(Exception):
    """Raise from subcommands to report error back to the user"""
